final_prep/mod_zero_hero.rb

```diff
@@ -1,84 +1,204 @@
-
-# Create a function called assess_situation that takes three arguments - danger_level, save_the_day, bad_excuse
-#   - danger_level should be an integer
-#   - save_the_day should be a string a hero would say once they save the day 
-#   - bad_excuse should be a string a hero would say if they are too afraid of the danger_level
+puts sidekicks
+
+# Create a function called assess_situation that takes three arguments -
+# danger_level, save_the_day, bad_excuse
+def assess_situation(danger_level, save_the_day, bad_excuse)
+  #   - danger_level should be an integer
+  danger_level = 9
+  #   - save_the_day should be a string a hero would say once they save the day
+  save_the_day = "My work here is done. You are safe now."
+  #   - bad_excuse should be a string a hero would say if they are too afraid of
+  #     the danger_level
+  bad_excuse = "It's too dangerous. We need more help!"
 
 # Your function should include an if/else statement that meets the following criteria
-#   - Danger levels that are above 50 are too scary for your hero. Any danger level that is above 50 should result in printing the bad_excuse to the terminal
-#   - Anything danger_level that is between 10 and 50 should result in printing the save_the_day string to the terminal
-#   - If the danger_level is below 10, it means it is not worth your time and should result in printing the string "Meh. Hard pass." to the terminal.
+#   - Danger levels that are above 50 are too scary for your hero. Any danger
+#     level that is above 50 should result in printing the bad_excuse to the terminal
+  if danger_level > 50
+    p bad_excuse
 
-#Test Cases
-announcement = 'Never fear, the Courageous Curly Bracket is here!'
-excuse = 'I think I forgot to lock up my 1992 Toyota Coralla. Be right back.'
-# assess_situation(99, announcement, excuse) > Should print - 'I think I forgot to lock up my 1992 Toyota Coralla. Be right back.'
-#assess_situation(21, announcement, excuse) > should print - 'Never fear, the Courageous Curly Bracket is here!'
-#assess_situation(3, announcement, excuse) > should print - "Meh. Hard pass."
+#   - Anything danger_level that is between 10 and 50 should result in printing
+#     the save_the_day string to the terminal
+  elsif danger_level >= 10 && danger_level <= 50
+    p save_the_day
 
-# Declare a new variable - scary_monster - assigned to an hash with the following key/values
-#   - name (string)
-#   - smell (string)
-#   - weight (integer)
-#   - citiesDestroyed (array)
-#   - luckyNumbers (array)
-#   - address (hash with following key/values: number , street , state, zip)
+#   - If the danger_level is below 10, it means it is not worth your time and
+#     should result in printing the string "Meh. Hard pass." to the terminal.
+  else
+    p "Meh. Hard pass."
+  end
 
+end
 
+# Test different danger levels and outputs
+p assess_situation("danger_level", "save_the_day", "bad_excuse")
```

Comment:
Here is where you would pass in those variables you declared above. For example:

```
danger_level = 9
save_the_day = "My work here is done. You are safe now."
bad_excuse = "It's too dangerous. We need more help!"
p assess_situation(danger_level, save_the_day, bad_excuse)
```